File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/org/openapitools/client/model/AuthenticationRequest.scala

package org.openapitools.client.model


case class AuthenticationRequest (
    /* The Mojang account e-mail or username. Never store it. */
    _username: String,
    /* The Mojang account password, never store it. */
    _password: String
)
object AuthenticationRequest {
    def toStringBody(var_username: Object, var_password: Object) =
        s"""
        | {
        | "username":$var_username,"password":$var_password
        | }
        """.stripMargin
}

File: generated-sources/php-slim/mojang-authentication/lib/Model/AuthenticationRequest.php
<?php
/**
 * AuthenticationRequest
 */
namespace OpenAPIServer\Model;

/**
 * AuthenticationRequest
 */
class AuthenticationRequest
{
    
    /** @var string $username The Mojang account e-mail or username. Never store it.*/
    private $username;
    
    /** @var string $password The Mojang account password, never store it.*/
    private $password;
}
